test/test_basic.py

```diff
@@ -181,18 +181,18 @@ def test_id_safety(request):
     with pytest.raises(ValueError, match="already"):
         v1.id = used_vol_id
 
-    safe_vol_id = 9876
+    safe_vol_id = 101
```

Comment:
For my own understanding.... this isn't part of the fix, right? Just circumstantial changes that came up in testing?  I'm fine to leave them, but want to make sure I understand.

Reply:
Turns out I added the wrong fix. Thanks for asking about this. I've updated it to include what I think is the correct fix along with comments on the intended tests.